tokens/color/surface.yaml

```diff
@@ -25,3 +25,40 @@ color:
     darkest:
       $value: '{color.gray.95}'
       $description: Primary surface (dark theme)
+
+    status:
```

Comment:
should these be `--rh-color-status-surface-` instead of `--rh-color-surface-status-`?